scalameta/semantic/shared/src/main/scala/scala/meta/semantic/Mirror.scala

```diff
@@ -33,24 +32,17 @@ object Mirror {
     *
     *                  Example:
     *                      "src/main/scala:build/Bar.scala".
-    * @param scalahostNscPluginPath File path to the scalahost-nsc compiler plugin fatjar,
```

Comment:
This is an amazing simplification. The complexity of this parameter was leaked onto all downstream users. I won't miss `q"Int".symbol`, however cute it was.